### ISSUE TITLE
Layout/IndentHeredoc issue on multi-line method calls

### ISSUE THREAD

**Issue by bquorning** [rubocop-hq/rubocop#7085]
When a squiggly heredoc is given on the second (or higher) line of a method argument, Layout/IndentHeredoc wants me to indent the heredoc two spaces more than the last line of method arguments. See this example code:

```rb
# frozen_string_literal: true

puts 'A very long argument that forces the 2nd argument onto the next line',
     'Here we go:', <<~TXT
  This is intended
  to be two-space indented.
  Did I find a bug?
TXT
```

RuboCop is happy (but I am not) if I indent like this:
```rb
# frozen_string_literal: true

puts 'A very long argument that forces the 2nd argument onto the next line',
     'Here we go:', <<~TXT
       This was intended
       to be two-space indented.
       Did I find a bug?
TXT
```

Please note that there are no offenses for the first example if I use `<<` or `<<-` instead of `<<~`.

--------

## Expected behavior

I would expect to have no offenses for the first example above.

## Actual behavior

```
Offenses:

test.rb:5:1: C: Layout/IndentHeredoc: Use 2 spaces for indentation in a heredoc.
  This was intended ...
^^^^^^^^^^^^^^^^^^^

1 file inspected, 1 offense detected
```

## Steps to reproduce the problem

See the first code example above.

## RuboCop version

```
0.70.0 (using Parser 2.6.3.0, running on ruby 2.6.3 x86_64-darwin18)
```

**Comment by bquorning** [comment]
I thought found a way to change the cop, but then it broke this untested, but valid indentation:

```rb
a_block do
  calling_a_method(
    :very_long_line,
    <<~RUBY
      foo
    RUBY
  )
end
```

**Comment by buehmann** [comment]
RuboCop is also happy with the following placement of the end marker. Just to give you another option …
```ruby
# frozen_string_literal: true

puts 'A very long argument that forces the 2nd argument onto the next line',
     'Here we go:', <<~TXT
       This was intended
       to be two-space indented.
       Did I find a bug?
     TXT
```

**Comment by bquorning** [comment]
The bug still exists on the most recent version
```
0.74.0 (using Parser 2.6.3.0, running on ruby 2.6.3 x86_64-darwin18)
```
